test/test_context.rb

```diff
@@ -57,6 +57,16 @@ def test_authenticate_with_token
     assert_logged_in(new_service)
   end
 
+  def test_authenticate_with_basic
+    new_arguments = @splunkrc.clone
+    new_arguments.delete(:username)
+    new_arguments.delete(:password)
+    new_arguments[:basic] = True
+
+    new_service = Context.new(new_arguments)
+    assert_logged_in(new_service)
```

Comment:
Can we also add something like `assert_true(service.apps.length() > 0)` to verify that the service can access a REST endpoint?